cmd/apiProviderUpload_test.go

```diff
@@ -66,6 +66,32 @@ func TestRunApiProviderUpload(t *testing.T) {
 		assert.EqualError(t, err, "HTTP POST request to /apiportal/api/1.0/Management.svc/APIProviders failed with error: : Bad Request")
 	})
 
+	t.Run("valid json file test", func(t *testing.T) {
+		properServiceKey := `{
+			"oauth": {
+				"url": "https://demo",
+				"clientid": "demouser",
+				"clientsecret": "******",
+				"tokenurl": "https://demo/oauth/token"
+				}
+			}`
+		apimData := apim.Bundle{Payload: properServiceKey}
+		assert.Equal(t, apimData.IsJSON(), true)
+	})
+
+	t.Run("invalid json file test", func(t *testing.T) {
+		properServiceKey := `{
```

Comment:
could you make it more clear that this is not json? It looks a lot like json to me. Also change the var name, as it is not proper, right?